specs/workflows/continue.spec.js

```diff
@@ -1,10 +1,11 @@
 import continueWorkflow from "../../src/workflows/continue";
-import * as run from "../../src/workflows/steps";
+import * as run from "../../src/workflows/steps/index";
```

Comment:
Instead of `import * as bla from "bla";` could you do... ` import { bla } from "bla";`?

Clear as mud?

Example...

```
import { gitStageFiles, gitRebaseContinue, getFeatureBranch } from "../../src/workflows/steps/indes";
import { verifyConflictResolution } from "../../src/workflows/steps/conflictResolution";
```